### ISSUE TITLE
App Apple Privacy Manifest to the SDK.

### ISSUE THREAD

**Issue by observableobject** [klaviyo/klaviyo-swift-sdk#170]
### Checklist

- [X] I have read the [contributing guidelines](https://github.com/klaviyo/klaviyo-swift-sdk/blob/master/.github/CONTRIBUTING.md)
- [X] This issue hasn't been addressed in an [existing issue](https://github.com/klaviyo/klaviyo-swift-sdk/issues) or [pull request](https://github.com/klaviyo/klaviyo-swift-sdk/pulls)

### Description

```markdown
Starting May 1st, Apple is requiring apps that contain SDKs that access required reason APIs to [include a bundled privacy manifest](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files) in the dependency. 



You need to include a privacy manifest file in your third-party SDK if it’s listed in “SDKs that require a privacy manifest and signature,” in Upcoming third-party SDK requirements. Otherwise, include a privacy manifest file in your third-party SDK if it uses required reasons API, collects data about the person using apps that include the third-party SDK, enables the app to collect data about people using the app, or contacts tracking domains. Providing a privacy manifest file helps app developers to understand the API use and data-collection practices of your third-party SDK.
```

### Proposed Solution

```markdown
If Klaviyo does NOT collect and privacy data or access on required reason APIs, this should be noted in the documentation and no manifest is necessary. 

A less than ideal solution that some other vendors have implemented is shipping the manifest as a separate file, linked in the docs, or (even less recommended) asking app developers to list the reasons in the App privacy manifest, which leads to inaccuracies if anything changes in the SDK. 

If Klaviyo accesses any privacy required reason APIs, please consider including a privacy manifest listing the reasons for accessing the required reason APIs.  Most other SDK vendors are putting this in `Sources/resources` folder. 

If Klaviyo collects any privacy data on users and transmits off device, please consider including a privacy manifest with the privacy nutrition labels.  Most other SDK vendors are putting this in `Sources/resources` folder.
```

### Alternatives Considered

```markdown
If Klaviyo does NOT collect and privacy data or access on required reason APIs, this should be noted in the documentation and no manifest is necessary. 

A less than ideal solution that some other vendors have implemented is shipping the manifest as a separate file, linked in the docs, or (even less recommended) asking app developers to list the reasons in the App privacy manifest, which leads to inaccuracies if anything changes in the SDK.
```

### Additional Context

```markdown
Please read the Apple documentation on the matter: 

https://developer.apple.com/documentation/bundleresources/privacy_manifest_files

https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api

https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests
```

**Comment by dotswift** [comment]
On my end, I cannot release my app until this SDK has the manifest. Love the work on `klaviyo-swift-sdk` so far!

**Comment by ajaysubra** [comment]
@observableobject @dotswift Thanks for filing this issue and providing all the context. We will look into this and get back to y'all soon.

**Comment by ajaysubra** [comment]
Hey, quick update - Cut out a PR [here](https://github.com/klaviyo/klaviyo-swift-sdk/pull/174) for this and will put out a release early next week. Thanks for your patience.

**Comment by ajaysubra** [comment]
Just published a release ([`3.1.0`](https://github.com/klaviyo/klaviyo-swift-sdk/releases/tag/3.1.0)) that among other things includes the privacy manifest in the SDK. Please let us know if you have any trouble with this. Thanks for your patience.
